x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/rule_migrations_task_client.ts

```diff
@@ -109,19 +114,30 @@ export class RuleMigrationsTaskClient {
   /** Returns the stats of a migration */
   public async getStats(migrationId: string): Promise<RuleMigrationTaskStats> {
     const dataStats = await this.data.rules.getStats(migrationId);
-    const status = this.getTaskStatus(migrationId, dataStats.rules);
-    return { status, ...dataStats };
+    const taskStats = this.getTaskStats(migrationId, dataStats.rules);
+    return { ...taskStats, ...dataStats };
   }
 
   /** Returns the stats of all migrations */
   async getAllStats(): Promise<RuleMigrationTaskStats[]> {
     const allDataStats = await this.data.rules.getAllStats();
     return allDataStats.map((dataStats) => {
-      const status = this.getTaskStatus(dataStats.id, dataStats.rules);
-      return { status, ...dataStats };
+      const taskStats = this.getTaskStats(dataStats.id, dataStats.rules);
+      return { ...taskStats, ...dataStats };
```

Comment:
Consider adding tests for this change.

Reply:
I missed this test. I'll create it in a separate PR

Reply:
in the end I added this test here https://github.com/elastic/kibana/pull/212799/commits/d448b3e0e73b8924f3577b5aa62c1baa5db4b040